ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -491,6 +491,12 @@ def __array__(self, dtype=None, /):
         if not dtype:
             return self
         return np_frontend.array(self, dtype=dtype)
+    
+    def __array_wrap__(self, array, context=None, /):
+        if context is None:
```

Comment:
When context is None that's when we return `np_frontend.array(array)`

Reply:
I think the changes you suggested are a good idea. They make the method more flexible. I'm happy to make these changes. They are passing all the tests locally. 
Thanks for your feedback.